http-server-jetty/src/main/java/io/micronaut/servlet/jetty/JettyFactory.java

```diff
@@ -167,6 +191,20 @@ protected Server jettyServer(
         return server;
     }
 
+    /**
+     * @param jettyRequestLog the jetty request log configuration.
+     * @return {@link RequestLog} logback-access impl.
+     */
+    @Singleton
+    @Requires(property = JettyConfiguration.JettyRequestLog.ENABLED_PROPERTY, value = StringUtils.TRUE)
+    RequestLog requestLog(JettyConfiguration.JettyRequestLog jettyRequestLog) {
+        RequestLogImpl requestLog = new RequestLogImpl();
```

Comment:
in addition I believe request access logs are supported in 3 different servlet containers so if logback-access is a Jetty only thing we need to document that

Reply:
I have added section in jetty docs about it.